src/components/Session/sagas.js

```diff
@@ -51,7 +55,13 @@ export function* createSessionRequest({ payload }) {
   const response = yield call(createSession, realmId, name, time, dollarsToCents);
 
   if (response.data) {
-    yield put(createSessionSuccess(response.data.putSession));
+    const { putSession } = response.data
+    yield put(createSessionSuccess(putSession));
+    const path = location.pathname.replace("new", putSession.id);
```

Comment:
I've actually implemented a library to dispatch actions which then get handled by react router. This case here prevents the browser from keeping history. There's an example over in https://github.com/tomarrell/poker_tracker_ui/blob/master/src/components/Login/sagas.js of the push method provided by react-router-redux

Reply:
@tomarrell I don't get what I'm doing different to the saga in Login? I based it on that.